cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/JepSingleton.kt

```diff
@@ -118,26 +122,18 @@ object JepSingleton {
                 virtualEnv = System.getenv("CPG_PYTHON_VIRTUALENV")
             }
 
+            val virtualEnvPath = "${System.getProperty("user.home")}/.virtualenvs/${virtualEnv}/"
+
             val wellKnownPaths =
                 listOf(
-                    File(
-                        "${System.getProperty("user.home")}/.virtualenvs/${virtualEnv}/lib/python3.11/site-packages/jep/libjep.so"
-                    ),
-                    File(
-                        "${System.getProperty("user.home")}/.virtualenvs/${virtualEnv}/lib/python3.11/site-packages/jep/libjep.jnilib"
-                    ),
-                    File(
-                        "${System.getProperty("user.home")}/.virtualenvs/${virtualEnv}/lib/python3.10/site-packages/jep/libjep.so"
-                    ),
-                    File(
-                        "${System.getProperty("user.home")}/.virtualenvs/${virtualEnv}/lib/python3.10/site-packages/jep/libjep.jnilib"
-                    ),
-                    File(
-                        "${System.getProperty("user.home")}/.virtualenvs/${virtualEnv}/lib/python3.9/site-packages/jep/libjep.so"
-                    ),
-                    File(
-                        "${System.getProperty("user.home")}/.virtualenvs/${virtualEnv}/lib/python3.9/site-packages/jep/libjep.jnilib"
-                    ),
+                    File("${virtualEnvPath}/lib/python3.12/site-packages/jep/libjep.so"),
```

Comment:
can we extract this into a set of version = setOf("3.11", "3.12"), etc. and then iterate of this version?

Reply:
I'm not the Kotlin expert here. Does f488e6a5b33230de6e543eee5a34ce4dfe9d55c8 work for you?

Reply:
Works